apps/docs/pages/guides/auth/social-login.mdx

```diff
@@ -47,9 +46,13 @@ Supabase supports a suite of social providers. Follow these guides to configure
   
 ## Provider Tokens
   
-Once the OAuth flow completes, Supabase Auth will sign your user in. You will receive a copy of the provider token used in the OAuth flow in case you need to use it further. For example, you can use the Google provider token to access Google APIs on behalf of your user.
+You can use the provider token and provider refresh token returned to make API calls to the OAuth provider. For example, you can use the Google provider token to access Google APIs on behalf of your user. 
+
+Supabase Auth does not manage refreshing the provider token for the user. Your application will need to use the provider refresh token to obtain a new provider token. If no provider refresh token is returned, then it could mean one of the following:
+* The OAuth provider does not return a refresh token
+* Additional scopes need to be specified in order for the OAuth provider to return a refresh token.
```

Comment:
For consistency:

```suggestion
Supabase Auth does not manage refreshing the provider token for the user. Your application will need to use the provider refresh token to obtain a new provider token. If no provider refresh token is returned, then it could mean one of the following:
- The OAuth provider does not return a refresh token
- Additional scopes need to be specified in order for the OAuth provider to return a refresh token.
```